mappings/net/minecraft/world/World.mapping

```diff
@@ -36,41 +159,156 @@ CLASS net/minecraft/class_18 net/minecraft/world/World
 		ARG 1 entityBoundingBox
 		ARG 2 fluidMaterial
 		ARG 3 entity
-	METHOD method_172 (Lnet/minecraft/class_54;IIII)V
+	METHOD method_171 canInteract (Lnet/minecraft/class_54;III)Z
+		ARG 1 player
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
+	METHOD method_172 extinguishFire (Lnet/minecraft/class_54;IIII)V
+		ARG 1 player
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+		ARG 5 direction
+	METHOD method_173 worldEvent (Lnet/minecraft/class_54;IIIII)V
+		ARG 1 player
+		ARG 2 event
+		ARG 3 x
+		ARG 4 y
+		ARG 5 z
+		ARG 6 data
+	METHOD method_174 countEntities (Ljava/lang/Class;)I
+		ARG 1 entityClass
+	METHOD method_175 collectEntitiesByClass (Ljava/lang/Class;Lnet/minecraft/class_25;)Ljava/util/List;
+		ARG 1 entityClass
+		ARG 2 box
 	METHOD method_176 getOrCreateState (Ljava/lang/Class;Ljava/lang/String;)Lnet/minecraft/class_58;
 		ARG 1 stateClass
 		ARG 2 id
+	METHOD method_177 getPlayer (Ljava/lang/String;)Lnet/minecraft/class_54;
+		ARG 1 name
 	METHOD method_178 addParticle (Ljava/lang/String;DDDDDD)V
+		ARG 1 particle
+	METHOD method_179 playMusic (Ljava/lang/String;III)V
+		ARG 1 music
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
 	METHOD method_180 setState (Ljava/lang/String;Lnet/minecraft/class_58;)V
 		ARG 1 id
 		ARG 2 state
+	METHOD method_181 processBlockUpdates (Ljava/util/Collection;)V
+		ARG 1 blockUpdates
+	METHOD method_182 addEntities (Ljava/util/List;)V
+		ARG 1 entities
+	METHOD method_183 addEventListener (Lnet/minecraft/class_59;)V
+		ARG 1 eventListener
+	METHOD method_184 spawnGlobalEntity (Lnet/minecraft/class_57;)Z
+		ARG 1 entity
+	METHOD method_185 broadcastEntityEvent (Lnet/minecraft/class_57;B)V
+		ARG 1 entity
+		ARG 2 event
+	METHOD method_186 getClosestPlayer (Lnet/minecraft/class_57;D)Lnet/minecraft/class_54;
+		ARG 1 entity
+		ARG 2 range
+	METHOD method_187 explode (Lnet/minecraft/class_57;DDDF)Lnet/minecraft/class_60;
+		ARG 1 source
+		ARG 2 x
+		ARG 4 y
+		ARG 6 z
+		ARG 8 power
+	METHOD method_188 explode (Lnet/minecraft/class_57;DDDFZ)Lnet/minecraft/class_60;
+		ARG 1 source
+		ARG 2 x
+		ARG 4 y
+		ARG 6 z
+		ARG 8 power
+		ARG 9 fire
 	METHOD method_190 getEntityCollisions (Lnet/minecraft/class_57;Lnet/minecraft/class_25;)Ljava/util/List;
+		ARG 1 entity
+		ARG 2 box
 	METHOD method_191 playSound (Lnet/minecraft/class_57;Ljava/lang/String;FF)V
-	METHOD method_200 (IIII)Z
+		ARG 1 source
+		ARG 2 sound
+	METHOD method_192 (Lnet/minecraft/class_57;Lnet/minecraft/class_57;F)Lnet/minecraft/class_61;
+		ARG 1 entity
+	METHOD method_193 updateEntity (Lnet/minecraft/class_57;Z)V
+		ARG 1 entity
+	METHOD method_194 processScheduledTicks (Z)Z
+	METHOD method_195 saveDisplay (ZLnet/minecraft/class_62;)V
```

Comment:
Should this be renamed to `saveWithLoadingProgressDisplayed` to better indicate that this function actually saves the world and doesn't just bring up the save screen?

Reply:
will do